modules/ROOT/pages/map-users-roles-ldap-task.adoc

```diff
@@ -1,39 +1,152 @@
 
+You can use any identity providers that return groups claims on the `UserInfo` or `Token` response.
+
+=== Map OIDC Group Users to Roles
+
+. Obtain a response to the `UserInfo` or `Token` endpoint that contains a groups claim from your external identity provider.
+. In Anypoint Platform, navigate to Access Management.
+. Click *Identity Providers*.
+. From *Identity Providers*, select OIDC.
+. If your IdP requires an additional OIDC scope to authorize access to groups claims, enter it in the *Group Scope* field.
+. Verify that the JSONata query in the *Group Attribute JSONata Expression* yields an array of group names from either the `UserInfo` or `Token` response.
+. Click *Roles*.
+. Click *Add Role* to create a role for each group of users in your organization.
+. Specify a role name and description.
+. Click *Add Role* to add the role.
 . In the *Roles* section, click the name of the new role:
+. Click *Set external group mapping*.
+. Enter the group to which you want to map.
+. Click *Set names*.
 +
-image::external-identity-35f9a.png[]
-+
+To map more than one attribute name to the selected role, click *Add More* and add another attribute.
 
-. Click *Set external group mapping*:
-+
-image::external-identity-251b8.png[]
-+
+=== Examples of OIDC Group Expressions
+
+The following examples show responses from the `UserInfo` or `Token` endpoints that contain groups claims from different external identity providers.
+
+==== Okta
+
+The following example shows a typical Okta groups claim.
 
+[source,xml]
+----
+{
+ ...
+  "groups": [
+    "Everyone",
+    "groupOne"
+  ]
+}
+----
+
+Given that groups claim, the JSONata expression is `groups`.
+
+==== Auth0
+
+The following example shows a typical Auth0 groups claim.
+
+[source,xml]
+----
+{
+  ...
+  "https://anypoint.mulesoft.com/groups": [
+    "MyGroup",
+    "groupOne"
+  ]
+}
+----
+
+Given that groups claim, the JSONata expression is `https://anypoint.mulesoft.com/groups`.
+
+==== Salesforce
+
+The following example shows a typical Salesforce groups claim.
+
+[source,xml]
+----
+{
+  ...
+  "custom_attributes": {
+    "PermissionSets": "groupOne,groupTwo"
+  }
+}
+----
+
+Given that groups claim, the JSONata expression is `$split(custom_attributes.PermissionSets, ',')`.
+
+== Configure Multiple Group Mappings for Single Sign-On using LDAP
+
+You can obtain the information you need to map your LDAP group to a role using a SAML assertion. Note that user management through LDAP is available only for the Private Cloud Edition.
+
+=== Map LDAP Group Users to Roles
+
+. In Anypoint Platform, navigate to Access Management.
+. Click *Roles*.
+. Click *Add Role* to create a role for each group of users in your organization.
+. Specify a role name and description.
+. Click *Add Role* to add the role.
+. In the *Roles* section, click the name of the new role:
+. Click *Set external group mapping*:
 . Copy the string from your SAML assertion AttributeValue to the External Group Name, for example:
```

Comment:
Ditto previous edits to similar text above:
access management
one single set of steps or one repeated single set of step
colons: the final three steps end with colons? Suggest not ending narrative headings/titles with colons.